aasdk_proto/ButtonCodeEnum.proto

```diff
@@ -42,6 +42,14 @@ message ButtonCode
         PREV = 0x58;
         PLAY = 0x7E;
         PAUSE = 0x7F;
+        MUSIC = 0xD1;
         SCROLL_WHEEL = 65536;
+        MEDIA = 65537;
```

Comment:
Is there a reason why you have decimal values instead of hex values like the rest of them in this file?

Reply:
I just copied the constants like they were in the other project.

Reply:
I think perhaps, the better way to do this is to fix the one line (`SCROLL_WHEEL`) where it is inconsistent and add everything as hex.